src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -57,6 +58,9 @@ def ols_ramp_fit_multi(ramp_data, buffsize, save_opt, readnoise_2d, gain_2d, wei
         'half', and 'all'. This is the fraction of cores to use for multi-proc. The
         total number of cores includes the SMT cores (Hyper Threading for Intel).
 
+    avg_dark_current: float (electrons)
```

Comment:
I see `avg_dark_current` added here to the parameter list description, but it has NOT been added to the actual list of arguments in the function definition up at line 23. Is that because `avg_dark_current` has now been moved to be part of the data class and hence doesn't need to be passed as an argument? If so, it should be removed from this list here.

Also, the definition for `algorithm` can be removed from this parameter list, because it's no longer in use.

Reply:
I removed the two deprecated parameters. 
Yes, avg_dark_current is now in the ramp_data class.